app/src/app/store/reducers/app.reducer.ts

```diff
@@ -53,7 +53,7 @@ export function appReducer(state: AppState = INITIAL_APP_STATE, action: Action):
     case AppActionTypes.SetHeaderColor:
       newState = {
         ...state,
-        headerColor: (action as SetHeaderColorAction).color
+        headerColor: (action as SetHeaderColorAction).payload
```

Comment:
the property is renamed to `payload`, but the value is still the same, so not sure it changes the behaviour

Reply:
When using the Action/Effect effect from extension.json, this change was required for the header color to actually change.